create_object.yml

```diff
@@ -7,7 +7,8 @@
 # Usage:
 #
 #   $ bin/ansible-playbook create_object.yml \
-#       -e "object_template=templates/openshift/edxapp/job/import_demo_course.yml.j2"
+#       -e "object_template=templates/openshift/edxapp/job/import_demo_course.yml.j2" \
+#       -e "deployment_stamp=2018-06-07-15-41-31-838798865"
```

Comment:
I really think that we should get rid of the hash and that a stamp to the second is really all we need. It could be formatted like this for clarity:
180607-15h41m31s

Same for deployment stamps.

So we could prepend them with D for deployment and J for job : 

- D-180607-15h41m31s
- J-180607-15h41m33s

Reply:
yes for humans also it's better if it's as short as possible. I think you can make it 100% of the cases because a CI trying to deploy the same project twice in a second is probably mibehaving?

Reply:
Probably true. And in a blue/green deployment strategy, this "should not" impact the current route. But, you know, as a scientist, I am always doubting... :wink:

Reply:
Can you add a sugar script (e.g. `bin/launch_job patient0 staging import_demo_course`) who automaticaly calculate the stamp ?

Reply:
The `job_stamp` is already handled by the `create_object` playbook. We cannot guess the `deployment_stamp` as, given a particular customer and environment, we need to know which stack we are targeting, _i.e._ previous, next or current?

But adding a sugar script to ease job restart is a pretty good idea!

Reply:
@cchaudier I've added this script in my last fixup. Check this out!